### DISCUSSION TITLE
Custom tags in html

### DISCUSSION THREAD

**Discussion by firedev** [bridgetownrb/bridgetown#811] [Q&A] [answered]
Studying `bridgetown-website` I've noticed some custom tags without definitions: `<main-content`, `<section-wrapper>` and the like. 

What are those? What is their purpose?

**Comment by ayushn21** [comment, marked as answer]
Check out this blog post: https://www.bridgetownrb.com/showcase/custom-html-elements-everywhere-for-page-layout/ and then this one: https://www.bridgetownrb.com/showcase/custom-html-elements-everywhere-executing-the-plan/.

They should answer your question I think :)

**Reply by firedev** [reply]
Appreciate it.
